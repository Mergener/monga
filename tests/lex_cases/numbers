4
522
-323

400e-2
4.045
2147483647
2147483648
-2147483648