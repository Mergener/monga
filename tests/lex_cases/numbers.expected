lit_int 4
lit_int 522
-
lit_int 323
lit_float 4.000
lit_float 4.045
lit_int 2147483647
lit_int 2147483648
-
lit_int 2147483648

